src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectRelationshipDetails.java

```diff
@@ -18,11 +18,12 @@ public void addTo(PageBuilder pageBuilder) {
             .page("correctPaperCaseDetails", this::midEvent)
             .pageLabel("Correct paper case")
             .label("Label-CorrectYourApplication", "### Your application details")
+            .mandatoryWithLabel(CaseData::getSupplementaryCaseType,
```

Comment:
small picky point but can we add NA to the label?

Reply:
Done